src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="/"
+      method="POST"
+      data-qa="big"
+      class="search-field"
+    >
+      <label class="label-input">
```

Comment:
```suggestion
      <label class="label-input label-input-big">
```
and in `label-input-big` insert icon as ::before